.circleci/generate.sh

```diff
@@ -30,12 +30,13 @@ die ()
 
 print_help()
 {
-  echo "Usage: $0 [-f|-p|-a|-e|-i]"
+  echo "Usage: $0 [-f|-p|-a|-e|-i|-s]"
   echo "   -a Generate the config.yml, config.yml.FREE and config.yml.PAID expanded configuration"
   echo "      files from the main config_template.yml reusable configuration file."
   echo "      Use this for permanent changes in config that will be committed to the main repo."
   echo "   -f Generate config.yml for tests compatible with the CircleCI free tier resources"
   echo "   -p Generate config.yml for tests compatible with the CircleCI paid tier resources"
+  echo "   -s Skip auto injection of changed tests. Useful when you need to repeat a single one i.e."
```

Comment:
```suggestion
  echo "   -s Skip automatic detection of changed tests. Useful when you need to repeat a few ones,"
  echo "      or when there are too many changed tests for CircleCI."
```

Reply:
Note that there is a second line in my suggestion above mentioning another use case for the new `-s`, which is when there are too many tests for CircleCI. That's what we have found in large patches like the ones for Tries and SAI.